### DISCUSSION TITLE
Intrusive popups in `INSERT` mode

### DISCUSSION THREAD

**Discussion by adriangalilea** [helix-editor/helix#10583] [General] [answered]
<img width="1028" alt="image" src="https://github.com/helix-editor/helix/assets/90320947/c9941dc0-1e07-4b85-a9a0-a8cfb15d7eaf">

I don't know what's the name for the popup on the top but it makes it impossible to read the code, specially combined with the lsp suggestions, I tried removing all my `[editor.lsp]` options just in case it was something I enabled, but I don't know what it is.

**Comment by pascalkuthe** [comment, marked as answer]
That's the signature help. You can toggle it off by disabling the auto signature help setting. 

This signature help is just way too high to be useful. I haven't seen something like that before. That seems like an issue with yhr languageserver you are using

**Reply by adriangalilea** [reply]
```
[editor.lsp]
auto-signature-help = false
```
This did the trick, I have no idea how to tweak the LSP tho,  so either I have it off or on, not ideal, but works for now, thanks.

**Reply by pascalkuthe** [reply]
You should raise an issue about this kn their issue tracker. For example to trim long types as in your example

**Reply by adriangalilea** [reply]
You mean on the LSP issue tracker?

```
[[language]]
name = "typescript"
language-servers = [ "typescript-language-server", "eslint", "emmet-ls", "gpt" ]
formatter = { command = "prettier", args = [ "--parser", "typescript" ] }
auto-format = true
```

So I remove 1 by one until I know which one is generating the message and check what their issue tracker is, right?
